repository_service_tuf/cli/admin/sign.py

```diff
@@ -167,19 +166,16 @@ def sign(
     signature = _add_signature_prompt(root_md, key)
 
     ###########################################################################
-    # Send payload to the API and save it locally
-    payload = SignPayload(signature=signature.to_dict())
-    if save:
-        json.dump(asdict(payload), save, indent=2)  # type: ignore
-        console.print(f"Saved result to '{save.name}'")
-    elif signing_json_input_file:
-        with open("sign-payload.json", "w") as out_file:
-            json.dump(asdict(payload), out_file, indent=2)
+    # Send payload to the API and/or save it locally
 
-        console.print("Saved result to 'sign-payload.json'")
+    payload = SignPayload(signature=signature.to_dict())
+    if save or not api_server:
```

Comment:
Is it intended to only consider the server cli arg here and the setting? Seems inconsistent.